semgrep/semgrep/core_runner.py

```diff
@@ -183,10 +188,11 @@ async def _core_stderr_processor(
 
             # readline returns empty when EOF
             if not line_bytes:
+                self._stderr = "".join(stderr_lines)
```

Comment:
**thought**: we could skip calling "".join() after every line
Maybe it'd speed this up even more if we saved the lines on an instance attributed, and moved the "".join() call to the `@property`.

Reply:
Ah, i see, it does actually run only once, when we encounter EOF. Ignore this comment :D